client/src/components/DataFiles/DataFilesModals/DataFilesMkdirModal.js

```diff
@@ -4,15 +4,27 @@ import { useDispatch, useSelector, shallowEqual } from 'react-redux';
 import * as Yup from 'yup';
 import { Formik, Form } from 'formik';
 import FormField from '_common/Form/FormField';
+import { findSystemDisplayName } from 'utils/systems';
 import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import findProjectTitle from '../../../utils/projects';
 
 const DataFilesMkdirModal = () => {
   const dispatch = useDispatch();
+  const systemList = useSelector(state => state.systems.systemList);
+  const projectsList = useSelector(state => state.projects.listing.projects);
+  const isOpen = useSelector(state => state.files.modals.mkdir);
   const params = useSelector(
     state => state.files.params.FilesListing,
     shallowEqual
   );
-  const isOpen = useSelector(state => state.files.modals.mkdir);
+  const systemDisplayName = (() => {
+    switch (params.scheme) {
```

Comment:
DataFilesUploadModal would also need this as it currently uses `findSystemDisplayName`.  Maybe these 10 lines could be put in a utility function and used there as well.

This might be PR/scope creep.  If it is, ignore and we can follow up later in a separate ticket/PR.

Reply:
I don't think it's too much extra to add into this. I think it's a bit easer to fix now that you've caught it here instead of in our testing sessions. I'll get this added in!